web/src/containers/activity/EntryDetails.js

```diff
@@ -67,41 +70,35 @@ const EntryDetails = ({ activity, aKey, num }) => {
   );
 
   return (
-    <div
-      id={`activity-${aKey}`}
-      className={`activity--body activity--body__${
-        collapsed ? 'collapsed' : 'expanded'
-      } activity--body__${num === 1 ? 'first' : 'notfirst'}`}
-      ref={container}
-    >
-      <Review heading={title} headingLevel={4} editContent={editContent} />
-      <div className={collapsed ? 'visibility--print' : ''}>
-        <Overview aKey={aKey} />
-        <Goals aKey={aKey} />
-        <Schedule aKey={aKey} />
-        <Costs aKey={aKey} />
-        <ContractorResources aKey={aKey} />
-        <CostAllocate aKey={aKey} />
-        <StandardsAndConditions aKey={aKey} />
-        <Button variation="primary" onClick={() => setCollapsed(true)}>
-          Done
-        </Button>
+    <Provider value={{ activity, index }}>
+      <div
+        id={`activity-${aKey}`}
+        className={`activity--body activity--body__${
+          collapsed ? 'collapsed' : 'expanded'
+        } activity--body__${num === 1 ? 'first' : 'notfirst'}`}
+        ref={container}
+      >
+        <Review heading={title} headingLevel={4} editContent={editContent} />
+        <div className={collapsed ? 'visibility--print' : ''}>
+          <Overview />
+          <Goals aKey={aKey} />
```

Comment:
We'll be able to yank out `aKey={aKey}` from the rest of these as we migrate them over.